lib/Pipes/CMakeLists.txt

```diff
@@ -19,10 +19,29 @@ revng_add_library_internal(
 
 llvm_map_components_to_libnames(
   LLVM_LIBRARIES
+  AggressiveInstCombine
+  Analysis
+  AsmParser
+  BitWriter
+  CodeGen
   Core
+  Coroutines
+  Extensions
+  GlobalISel
   IPO
+  IRReader
+  InstCombine
+  Instrumentation
+  Linker
   MC
+  Object
+  Passes
+  Remarks
+  ScalarOpts
   Support
+  Target
+  TransformUtils
+  Vectorize
```

Comment:
Shouldn't we only import libraries for which we call `llvm::initialize$LIB`?
I see `Object`, `Linker`, `IRReader` which seem a bit suspicious.

Reply:
Done